README.md

```diff
@@ -149,6 +149,68 @@ We use this branching model in fuse-starter-java:  http://nvie.com/posts/a-succe
  - Search for the Organization 'Galatea'
  - Bind to 'starter-java' and accept
 
+
+## Spring Annotations:
+Annotations are used to provide supplemental information about a program. They allow you to direct the framework to do your bidding, taking control and overriding its defaults when needed.
+Annotations are quick, easy to use, and orders of magnitude faster than building the equivalent functionality yourself.
+
+For a detailed list of annotations check (https://lightrun.com/java/the-complete-list-of-spring-boot-annotations-you-must-know/)
+
+
+- Basic setup:
+  - @Bean:
+    - It is a method-level annotation. It is an alternative of XML <bean> tag. It tells the method to produce a bean to be managed by Spring Container.
```

Comment:
Let's add a definition of "bean" here

Reply:
Also mention that we prefer using the stereotype annotations rather than defining beans in config classes where possible.

Reply:
Good start, but what if we have a new joiner with no java experience? Maybe a link to https://www.baeldung.com/spring-bean would help

Reply:
ah, github wasn't showing me the latest changes -- looks good, but let's add the link anyway